
robot:
  ros__parameters:

    wheel_base: 0.5
    track_width: 0.4
    wheel_radius: 0.1


    max_linear_velocity: 1.0
    max_angular_velocity: 1.0
    max_linear_acceleration: 0.5
    max_angular_acceleration: 0.5


    control_frequency: 50.0
    cmd_vel_timeout: 0.5


    lidar_topic: /scan
    odom_topic: /odom
    imu_topic: /imu
    camera_topic: /camera


    base_frame: base_footprint
    odom_frame: odom
    map_frame: map
    lidar_frame: lidar_link
    camera_frame: camera_link